check_dnsserials

```diff
@@ -158,10 +166,26 @@ for server in ${slaves}; do
 	if [ "x${serial_slave}" = "x${serial}" ]; then
 		verbose "OK:  ${domain} ${server} ${serial}"
 		else
-			verbose "BAD: ${domain} ${server} ${serial_slave}"
-			fails=$((fails + 1))
-			failedServers+=(${server})
-			debug "Incrementing failed states to ${fails}"
+                        if [ -n "${tolerance}" ]; then
```

Comment:
You won't ever reach the `else` codepath. `[ -n "0" ]` is always true and `$tolerance` is always a number > 0 here.

Reply:
Fixed